res/field/encounters_ex/mt_coronet_lake.json

```diff
@@ -0,0 +1,537 @@
+{
+    "species": "SPECIES_FEEBAS",
+    "map_dimensions": [
+        228,
+        300
+    ],
+    "tiles": [
```

Comment:
**question**: Is this a bounding-box? Can we just compute it from a set of rectangle-definitions?

Reply:
Upon further investigation it appears that these tiles are essentially a result of `tile = (v14 * 32 * y) + x` where I am guessing v14 is the height of the map or something like that. Not sure why it is multiplied by 32. Either way, if I can debug v14 during runtime I could turn these tile numbers into (x,y) coordinates and then it would be easier to make a bounding box, if possible.